services/businessconfig/src/main/modeling/swagger.yaml

```diff
@@ -114,6 +114,28 @@ paths:
           description: Not found  
         '500':
           description: Unable to delete process
+  '/businessconfig/processes/history/{processId}':
```

Comment:
If we have a process called 'history', it can be a problem I think. What do you think of this endpoint /businessconfig/processhistory/{processId} ?